app/src/services/auth-services.ts

```diff
@@ -53,24 +53,19 @@ export default class AuthServices {
    * Throws on error.
    */
   static async fetchUserInfo (): Promise<any> {
-    // PREVENT_STORAGE_SYNC flag is set during signout in sbc common components
-    // when session is state cleared, as a mechanism to prevent returning async
-    // functions to set tokens
-    if (sessionStorage.getItem('PREVENT_STORAGE_SYNC')) return null
```

Comment:
Code seems to work without console messages after logging in, logging out, logging in again, and logging out again.

Reply:
I tried to reproduce the original issue locally, and could not as well. The check was put in place to take advantage of the flag in the logout function: https://github.com/bcgov/sbc-common-components/blob/d640dc75ea51add2d611a30259d15d93d8654381/vue/sbc-common-components/src/services/keycloak.services.ts#L112 -- this would guard against race conditions in async contexts, It might be possible to reproduce locally if a delay is added to the logout function, e.g. smth along the line:

```
async logout(redirectUrl?: string) {
  let token = ConfigHelper.getFromSession(SessionStorageKeys.KeyCloakToken) || undefined;
  if (token) {
    this.kc = Keycloak(ConfigHelper.getKeycloakConfigUrl());
    let kcOptions: KeycloakInitOptions = {
    ...
    };
    const siteminderLogoutUrl = ConfigHelper.getFromSession(SessionStorageKeys.SiteminderLogoutUrl) || undefined;
    
    // Simulate delay before clearing session
    setTimeout(() => {
      ConfigHelper.clearSession();  // Clear session after delay
      ConfigHelper.addToSession(SessionStorageKeys.PreventStorageSync, true);
    }, 1000); // Delay in milliseconds
    
    return new Promise<void>((resolve, reject) => {
      this.kc && this.kc.init(kcOptions)
        .then(authenticated => {
          ...
        });
    });
  }
}

```

Reply:
Thanks for looking into this, Andrew.

Were you able to demonstrate that the original issue still exists?

Is there any code that removed the session storage variable, PREVENT_STORAGE_SYNC?

Why block the call to fetch user info? Is there anything else that could be done instead, so that the user info feature keeps working correctly? (See ticket for bug info.)